documentation/oc.asciidoc

```diff
@@ -0,0 +1,17 @@
+:toc:
+toc::[]
+
+= OpenShift CLI
+
+The oc commandlet is used to install OpenShift CLI. The instructions on how to use it can be found https://docs.openshift.com/container-platform/4.7/cli_reference/openshift_cli/getting-started-cli.html[here].
```

Comment:
Maybe use this sentence:
```
Call `devon oc «args»` with the arguments described https://docs.openshift.com/container-platform/latest/cli_reference/openshift_cli/getting-started-cli.html#cli-using-cli_cli-developer-commands[here].
```

Use also this link with latest and jump to the usage part.